### PR TITLE
fix: gaia profile test

### LINKED ISSUE #5072: Fix profile test
This test still seems to be a problem, though it passes locally. I believe there are multiple requests happening in this test and we are only handling one of them.

In this PR I skipped the test: https://github.com/leather-wallet/extension/pull/5063

Assigning to myself to fix now, just wanted an issue to track it.

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of Leather — [Extension build](https://github.com/leather-wallet/extension/actions/runs/8286447678), [Test report](https://leather-wallet.github.io/playwright-reports/fix/gaia-profile-test)

Catching the route to gaia and mocking the profile response in hopes of fixing the profile test. 🤞

### CODE REVIEW

**Review by pete-watters — APPROVED** [review]

File: tests/specs/profile/profile.spec.ts [hunk 1]
```diff
@@ -50,19 +61,30 @@ test.describe('Gaia request', () => {
...
     await accountsPage.close();
-  });
 
-  test.skip('should send a signed profile token to gaia', async ({ context }) => {
     await testAppPage.clickUpdateProfileButton();
     const profileUpdatingPage = new UpdateProfileRequestPage(await context.waitForEvent('page'));
 
     const requestPromise = profileUpdatingPage.page.waitForRequest('https://hub.blockstack.org/*');
 
+    await profileUpdatingPage.page.route('https://gaia.hiro.so/hub/*', async route => {
```
Comment (pete-watters) [inline tests/specs/profile/profile.spec.ts:62]: Maybe we can use this approach to add some mocks for Ordinals so we can test the send BRC20 / ordinal flow? 

I need to do that soon as it suprised me again on containers. I have [this task](https://github.com/leather-wallet/extension/issues/4467) for a while

File: tests/specs/profile/profile.spec.ts [hunk 2]
```diff
@@ -13,14 +12,19 @@ test.describe('Profile updating', () => {
     await onboardingPage.signInWithTestAccount(extensionId);
     testAppPage = await TestAppPage.openDemoPage(context);
     await testAppPage.signIn();
+  });
+
+  test.describe.configure({ mode: 'serial' });
```
Comment (kyranjamie) [inline tests/specs/profile/profile.spec.ts]: With the mocking is serial necessary?

File: tests/specs/profile/profile.spec.ts [hunk 3]
```diff
@@ -50,19 +61,30 @@ test.describe('Gaia request', () => {
...
+      timeout: 30000,
+    });
     await accountsPage.close();
-  });
 
-  test.skip('should send a signed profile token to gaia', async ({ context }) => {
     await testAppPage.clickUpdateProfileButton();
     const profileUpdatingPage = new UpdateProfileRequestPage(await context.waitForEvent('page'));
 
     const requestPromise = profileUpdatingPage.page.waitForRequest('https://hub.blockstack.org/*');
```
Comment (kyranjamie) [inline tests/specs/profile/profile.spec.ts:60]: Does this trigger at all? I half-looked into this and couldn't get this to log

**Review by kyranjamie — APPROVED** [review]
Perfect, thanks for fixing.

File: tests/specs/profile/profile.spec.ts (hunk 3 above)
Comment (fbwoolf) [reply tests/specs/profile/profile.spec.ts:60]: I did see the log when using `--ui` flag running tests in UI mode.

File: tests/specs/profile/profile.spec.ts (hunk 2 above)
Comment (fbwoolf) [reply tests/specs/profile/profile.spec.ts]: I can try removing it.

**Pull request merged**